deploy/operator/common.sh

```diff
@@ -25,14 +25,21 @@ export HIVE_NAMESPACE="${HIVE_NAMESPACE:-hive}"
 # Versions #
 ############
 
+export UPGRADE_ASSISTED_OPERATOR="${UPGRADE_ASSISTED_OPERATOR:-false}"
 export OPENSHIFT_VERSIONS="${OPENSHIFT_VERSIONS:-$(cat ${__root}/data/default_ocp_versions.json)}"
 
 if [ -z "${ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE:-}" ]; then
-    export ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE=$(echo ${OPENSHIFT_VERSIONS} | jq -rc '[.[].release_image]|max')
+    if [ "${UPGRADE_ASSISTED_OPERATOR}" = "true" ]; then
+        export ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE=$(echo ${OPENSHIFT_VERSIONS} | jq -rc '[.[].release_image][-2]')
```

Comment:
why `-2`? I don't understand these code section. In any case, we must not depend on indexing that might get change from time to time. (e.g. 4.6 and 4.7 will soon be removed)

Reply:
Before the upgrade we are looking to install the version prior to the most current one of OCP. The most current version of OCP we are installing is 4.9, and the version previous to that is 4.8.

TLDR of the current flow is:
- Install hub cluster
- Install operator
- Install OCP 4.8 SNO spoke cluster
- Update operator
- Install OCP 4.9 SNO spoke cluster

Reply:
Thank you, Lisa. I can add comments explaining the logic but I don't see any other alternative atm.

Reply:
Added comments in the script explaining the need and logic.

Reply:
Thank you for the clarification